src/main/java/frc/robot/Constants.java

```diff
@@ -1,11 +1,24 @@
 package frc.robot;
 
 public class Constants {
+    public class DriveConstants{
+        public static final int DRIVE_CURRENT_LIMIT = 40;
 
-    public static final int FRONT_LEFT_ID = 20;
-    public static final int FRONT_RIGHT_ID = 21;
-    public static final int BACK_LEFT_ID = 22;
-    public static final int BACK_RIGHT_ID = 23;
-    public static final int DRIVE_CURRENT_LIMIT = 40;
+        // Motor IDs
+        public static final int BACKLEFT_ID = 20;
+        public static final int BACKRIGHT_ID = 21;
+        public static final int FRONTLEFT_ID = 22;
+        public static final int FRONTRIGHT_ID = 23;
+    }
     
+    public class CandyDispenser{
+        public static final int DISPENSER_ID = 21;
+        public static final int DISPENSER_CURRENT_LIMIT = 40;
+        public static final double kP = 0.5;
```

Comment:
These don't look like proper values.  kI is usually 0, since we don't use it, and I don't know how much difference kV and kA will make for the candy dispenser since it is so light.